src/services/sports/index.jsx

```
@@ -0,0 +1,400 @@
+            className="w-full h-full object-contain"
+          />
+        </div>
+
+        {/* Action Buttons */}
+        {!isTop && (
+          <div className="action-buttons-container">
+            <button
+              onClick={handleLike}
+              className="action-button"
+              disabled={isLoading}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)